AutomatedTesting/Gem/PythonTests/scripting/ScriptEvents_AllParamDatatypes_CreationSuccess.py

```diff
@@ -138,30 +139,17 @@ async def run_test(self):
 
         # 7) Set different names and datatypes for each parameter
         tools.expand_qt_container_rows(self, NAME_STRING)
-        children = self.asset_editor_row_container.findChildren(QtWidgets.QFrame, NAME_STRING)
-        index = 0
-        for child in children:
-            line_edit = child.findChild(QtWidgets.QLineEdit)
-            if line_edit is not None and line_edit.text() == "ParameterName":
-                line_edit.setText(f"param_{index}")
-                index += 1
+        # give all name fields a unique name using index variable
+        self.set_name_field_names()
 
-        children = self.asset_editor_row_container.findChildren(QtWidgets.QFrame, "Type")
-        index = 0
-        for child in children:
-            combo_box = child.findChild(QtWidgets.QComboBox)
-            if combo_box is not None and index < N_VAR_TYPES:
-                combo_box.setCurrentIndex(index)
-                index += 1
+        # give all the type fields a different type using the index for combo box selection
+        self.set_type_field_types()
```

Comment:
If a comment is needed to say what this function call is doing either the name of the function and the docstrings for the function need to be improved, or the comments are redundant and need to be removed.

Reply:
removed the comments. This part of the test script looks much cleaner now